rs/tests/common.bzl

```
@@ -310,9 +310,7 @@ UNIVERSAL_VM_RUNTIME_DEPS = [
     "//rs/tests:create-universal-vm-config-image.sh",
 ]
 
-GRAFANA_RUNTIME_DEPS = UNIVERSAL_VM_RUNTIME_DEPS + [
-    "//rs/tests:grafana_dashboards",
-]
+GRAFANA_RUNTIME_DEPS = UNIVERSAL_VM_RUNTIME_DEPS + [] # The dashboards are being pulled from k8s repository
```

I wouldn't mention the old situation anymore. 
```suggestion
GRAFANA_RUNTIME_DEPS = UNIVERSAL_VM_RUNTIME_DEPS
```